src/panel/poi/blocks/StarsSummary.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { capitalizeFirst } from 'src/libs/string';
+import { useI18n } from 'src/hooks';
+
+const StarsSummary = ({ block }) => {
+  const { _, _n } = useI18n();
+
+  if (!(block?.ratings?.[0]?.has_stars === 'yes')) {
+    return null;
+  }
+
+  if (block.ratings[0].nb_stars && block.ratings[0].nb_stars > 0) {
+    return (
+      <span>
+        {' · ' + capitalizeFirst(_n('%d star', '%d stars', block.ratings[0].nb_stars, 'poi'))}
+      </span>
+    );
+  }
+
+  return <span>{' · ' + capitalizeFirst(_('Starred', 'poi'))}</span>;
+};
+
+export default StarsSummary;
```

Comment:
I'm not how I would do that but the structure of this code is quite redundant, maybe the string could be built from a specific function that would take a "separator" parameter which indicates if we want it to be full-text or with a middle dot. Do we know if this difference of behavior is actually willingly pushed by the design?